tsconfig.json

```diff
@@ -4,6 +4,11 @@
   "compilerOptions": {
     "baseUrl": "./",
     "paths": {
+      "@skyux/colorpicker": [
+        "node_modules/@skyux/colorpicker",
+        "dist/colorpicker",
+        "projects/colorpicker/src/public-api.ts"
```

Comment:
This is my fault, but we only need to list the `projects` folder in this array.

Reply:
Done.